cmd/immutest/command/init.go

```diff
@@ -29,8 +29,8 @@ import (
 	"github.com/codenotary/immudb/pkg/api/schema"
 	"github.com/codenotary/immudb/pkg/auth"
 	"github.com/codenotary/immudb/pkg/client"
-	"github.com/codenotary/immudb/pkg/gw"
 	"github.com/codenotary/immudb/pkg/server"
+	"github.com/codenotary/immugw/pkg/gw"
```

Comment:
idem with immuadmin, it might be a signal of something that needs to be refactored

Reply:
Fixed